### PR TITLE
API docs review

### PR DESCRIPTION
**Pull request by edstros**
Edited to correct grammar, validate displayed json, fix broken links, clean up code blocks in tables (by reformatting tables). Most recent changes can be found here: https://github.com/edstros/phy-api-docs/pull/1/files

### CODE REVIEW

**Comment by edstros** [comment]
Hold on, I missed a table. Will update when finished.

**Comment by edstros** [comment]
OK, went through everything once more and this should be good to go. Quite a good exercise, actually. Thanks for the opportunity, @rochforp.

**Comment by kevinahuber** [comment]
- [ ] Add table of contents
- [ ] Change developer resource link to `https://help.phyplatform.support/hc/en-us/categories/115000680507-Developer-Resources`
- [ ] Remove link to wiki

**Comment by edstros** [comment]
[x] Add table of contents -- README.md page is updated with TOC (incorporating the content from Home.md, which was subsequently deleted).
[x] Change developer resource link to `https://help.phyplatform.support/hc/en-us/categories/115000680507-Developer-Resources` -- Developer resource link is changed to the Knowledge Base link.
[x] Remove link to wiki -- Link to wiki removed.

**Comment by kevinahuber** [comment]
- [ ] `Enterprise` should read `Professional`
- [ ] `PHY.net` should read `Phy Platform`
- [ ] `TapAction` should read `TapActions`

**Comment by edstros** [comment]
@kevinahuber Language edits made to README and updated links from wiki to individual pages.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,18 @@
 # API Documentation for PHY.net
```
Comment (kevinahuber) [inline README.md]: This still says PHY.net.. should be Phy Platform

**Pull request merged**